micro_manager/interpolation.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from sklearn.neighbors import NearestNeighbors
+
+
+class Interpolation:
+    def __init__(self, logger):
+
+        self._logger = logger
+
+    def get_nearest_neighbor_indices_local(
```

Comment:
```suggestion
    def get_nearest_neighbor_indices(
```
Unless we have a global variant of the same function, no need to specify `local` in the function name. We can just go with a generic function name and specify the local indices part in the docstring.